fiware-region-sanity-tests/tests/fiware_region_object_storage_tests.py

```diff
@@ -51,25 +54,114 @@ def test_create_get_and_delete_container(self):
         self.assertEquals(len(response[-1]), 0)  # The list of items should be 0.
         self.logger.debug("Getting %s container details from the object storage" % containerName)
 
+    def test_delete_container(self):
+        """
+        Test if it can be possible delete a container.
+        """
+        suffix = datetime.utcnow().strftime('%Y%m%d%H%M%S')
+        containerName = TEST_CONTAINER_PREFIX + suffix
+
+        response = self.swift_operations.create_container(containerName)
+        self.assertIsNone(response)
```

Comment:
Add a message error for each assert, in order to see a comprehensible reason of problems in result reports.